draft.py

```diff
@@ -258,4 +271,75 @@ def run_current_lot(self):
       yield time_remaining
     
     self.rotateCaptainList()
-    self.machine.nom_from_bid()
\ No newline at end of file
+    self.machine.nom_from_bid()
+
+  def checkNum(self, number):
+    if type(number) == float:
+      return True
+    return False
+  
+  def player(self, message):
+    message_parts = message.content.split()
+    player_name = message_parts[1]
+    if self.is_admin(message):
+      if self.checkPlayer(player_name):
+        raise AuctionValidationError(ClientMessage(
+          type=ClientMessageType.CHANNEL_MESSAGE,
+          data=f"Can't add {player_name}, that player is already in the system."
+          ))
+      if len(message_parts) > 2:
+        player_mmr = float(message_parts[2])
+        if not self.checkNum(player_mmr):
+          raise AuctionValidationError(ClientMessage(
+            type=ClientMessageType.CHANNEL_MESSAGE,
+            data=f"Can't add {player_name}, improper MMR formatting."
+            ))
+        if not self.addPlayer(player_name, player_mmr):
```

Comment:
so this just doesn't work, right? you added a bunch of new arguments to the `addPlayer` method that you're no longer passing in?